### PR TITLE
CUP civilian vehicles pack

### PR DESCRIPTION
**Pull request by wersal454**
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:

Added civilian vehicles from CUP to AddonVics
    
1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Running around cities?
********************************************************
Notes:
!!!!
 requiredAddons[] = {"CUP_Vehicles"}; is 99% wrong, I just don't know what to wright here.

### CODE REVIEW

File: A3A/addons/core/Templates/AddonVics/cup_veh_Civ.sqf [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+	,"CUP_I_V3S_Refuel_TKG" , 0.1
+]];
+
+_addon set ["vehiclesCivBoat", [
+	"CUP_C_Fishing_Boat_Chernarus" , 0.2
+	,"CUP_C_Zodiac_CIV" , 1.0
+	,"CUP_C_PBX_CIV" , 1.0
+]];
+
+//_addon set ["vehiclesCivHeli", [    //planes for test
```
Comment (Lazejun) [inline A3A/addons/core/Templates/AddonVics/cup_veh_Civ.sqf:103]: No need to add Dead code

**Review by Lazejun — CHANGES_REQUESTED** [review]

File: A3A/addons/core/Templates/AddonVics/cup_veh_Civ.sqf (hunk 1 above)
Comment (Bob-Murphy) [reply A3A/addons/core/Templates/AddonVics/cup_veh_Civ.sqf]: done

**Review by Bob-Murphy — COMMENTED** [review]
Please check if the vehicles which could load cargo are defined in logistics/Nodes/CUP.hpp and define the ones which are not covered.

**Review by killerswin2 — APPROVED** [review]

**Comment by wersal454** [comment]
Hello there! so you merged my PR, but did you test it? Is it working like it should? For example: playing with vanilla factions but additionally civis will use vehicles from CUP and vanilla vehicles. Problem originally was that CUP vehicles just won't spawn no matter what. So if you fixed it I would love to know how so me or guys from Ultimate can fix similar PR there.

**Comment by jaj22** [comment]
In future do say if your PR doesn't work. PRs are generally assumed to run, and reviewers will be looking for other things.

**Pull request merged**